dataclients/kubernetes/kube_test.go

```diff
@@ -3617,3 +3626,99 @@ func TestCreateEastWestRouteOverwriteDomain(t *testing.T) {
 		})
 	}
 }
+
+func TestSkipperDefaultFilters(t *testing.T) {
```

Comment:
I miss a test, that does not apply the default filters to optionally created redirects.
@aryszka what do you think should this be applied to redirects?